pkgs/applications/misc/kemai/default.nix

```diff
@@ -0,0 +1,46 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  cmake,
+  magic-enum,
+  spdlog,
+  qt6,
+  xorg,
+  nix-update-script
+}:
+
+stdenv.mkDerivation rec {
+  version = "0.9.2";
+  pname = "kemai";
+
+  src = fetchFromGitHub {
+    sha256 = "sha256-PDjNO2iMPK0J3TSHVZ/DW3W0GkdB8yNZYoTGEd2snac=";
+    rev = version;
+    repo = "kemai";
+    owner = "AlexandrePTJ";
+  };
+
+  buildInputs = [
+    qt6.qtbase
+    qt6.qtconnectivity
+    qt6.qttools
+    qt6.qtlanguageserver
+    xorg.libXScrnSaver
+    magic-enum
+    spdlog
+  ];
+  cmakeFlags = [ "-DUSE_CONAN=OFF" ];
+  patches = [ ./cmake.patch ];
```

Comment:
What this diff does?

Reply:
Give a more descriptive name to it, something like `000-cmake-disable-conan.diff`

Also, as a matter of convention, `.patch` file extension is for mail-like commented patches (those including a header with authorship and description on the body before the diff lines), whereas `.diff` is for raw diff files.